app/javascript/packs/document-capture.tsx

```diff
@@ -14,47 +14,28 @@ import {
 import { isCameraCapableMobile } from '@18f/identity-device';
 import { FlowContext } from '@18f/identity-verify-flow';
 import { trackEvent as baseTrackEvent } from '@18f/identity-analytics';
-
-/** @typedef {import('@18f/identity-document-capture').FlowPath} FlowPath */
-/** @typedef {import('@18f/identity-i18n').I18n} I18n */
-
-/**
- * @typedef LoginGov
- *
- * @prop {Record<string,string>} assets
- */
-
-/**
- * @typedef LoginGovGlobals
- *
- * @prop {LoginGov} LoginGov
- */
-
-/**
- * @typedef {typeof window & LoginGovGlobals} DocumentCaptureGlobal
- */
```

Comment:
Nice thing about native TypeScript is that the unused values are flagged, unlike JSDoc-based types.